src/server.js

```diff
@@ -33,7 +34,7 @@ app.use("/api/homeUser", isAuthenticated, isHomeUser, homeUserRoute)
 app.use("/api/externalUser", isAuthenticated, isExternalUser, externalUserRoute)
 
 app.use("/api/smartHome", smartHomeRoutes)
-
+app.use("/api/smartHome", isAuthenticated, isHomeUser, modes)
```

Comment:
the `smartHome` route with authentication should have a different name from the one above, where you register/add the home

Reply:
probably best to add it to another file